### PR TITLE
CIV-6122 Update default case management location for Civil claims

### PR DESCRIPTION
**Pull request by asthamalviya**
https://tools.hmcts.net/jira/browse/CIV-6122

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/civil/referencedata/LocationRefDataService.java [hunk 1]
```diff
@@ -33,6 +33,56 @@ public class LocationRefDataService {
...
+            } else {
+                if (ctscLocations.size() > 1) {
+                    log.warn("Location Reference Data Lookup returned more than one CTSC location for Salford CTSC");
+                }
+                return ctscLocations.get(0);
+            }
+        } catch (Exception e) {
+            log.error("Location Reference Data Lookup Failed for Salford CTSC - " + e.getMessage(), e);
+        }
+        return LocationRefData.builder().build();
```
Comment (sabahirfan) [inline src/main/java/uk/gov/hmcts/reform/civil/referencedata/LocationRefDataService.java:58]: if lookup fails, is this valid to have empty Location ?

File: src/main/java/uk/gov/hmcts/reform/civil/referencedata/LocationRefDataService.java [hunk 2]
```diff
@@ -33,6 +33,56 @@ public class LocationRefDataService {
...
+            } else {
+                if (ctscLocations.size() > 1) {
+                    log.warn("Location Reference Data Lookup returned more than one CTSC location for Stoke CTSC");
+                }
+                return ctscLocations.get(0);
+            }
+        } catch (Exception e) {
+            log.error("Location Reference Data Lookup Failed for Stoke CTSC - " + e.getMessage(), e);
+        }
+        return LocationRefData.builder().build();
```
Comment (sabahirfan) [inline src/main/java/uk/gov/hmcts/reform/civil/referencedata/LocationRefDataService.java:83]: if lookup fails, is this valid to have empty Location ?

File: src/main/java/uk/gov/hmcts/reform/civil/referencedata/LocationRefDataService.java (hunk 1 above)
Comment (asthamalviya) [reply src/main/java/uk/gov/hmcts/reform/civil/referencedata/LocationRefDataService.java:58]: I have rewritten the method written by GA as getCcmccLocation with new locations. We are not using them anywhere in our civil-service code so not sure why they want it like that.

Comment (sabahirfan) [reply src/main/java/uk/gov/hmcts/reform/civil/referencedata/LocationRefDataService.java:58]: I think the common service should ideally throw some custom exception Not found etc. then based on the usage requirement user can ignore the error or rethrow this

Comment (asthamalviya) [reply src/main/java/uk/gov/hmcts/reform/civil/referencedata/LocationRefDataService.java:58]: @deepthidoppalapudihmcts Please can you confirm for Sabah's comment if we are ok with empty location. Thank you

**Review by dtortolaV1 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/reform/civil/referencedata/LocationRefDataService.java (hunk 1 above)
Comment (deepthidoppalapudihmcts) [reply src/main/java/uk/gov/hmcts/reform/civil/referencedata/LocationRefDataService.java:59]: @asthamalviya I agree with Sabah to throw Appropriate exception

**Review by MMNycz — APPROVED** [review]

**Pull request closed without merging**